.github/workflows/publish_connlib.yml

```diff
@@ -17,29 +20,16 @@ jobs:
         with:
           java-version: '17'
           distribution: 'adopt'
-      - uses: actions/cache@v3
+      - uses: Swatinem/rust-cache@v2
```

Comment:
Just noting for future reference, probably want some level of supply chain security applied to our GH actions dependencies.

@AndrewDryga might have input on our approach here, @bmanifold too when he starts.

Reply:
In rust-libp2p, we reference all 3rd party actions by hash with a comment next to it which version that is:

https://github.com/libp2p/rust-libp2p/blob/85a846a7dd5b54649114babde56dd7f2ec8e760b/.github/workflows/ci.yml#L40

Dependabot will pick that up and bump the reference automatically and update the version in the comment. Works well for us.

See https://github.com/libp2p/rust-libp2p/pull/3965/files for example.

Reply:
Ah, yeah that looks like it could work. Added #1687 to track this in another issue to prevent this PR from being blocked any further